lib/features/public_asset/data/network/public_asset_api.dart

```diff
@@ -88,15 +89,24 @@ class PublicAssetApi {
       .build()
       .execute();
     final publicAsset = response.parse<SetPublicAssetResponse>(
-        invocation.methodCallId,
-        SetPublicAssetResponse.deserialize
+      invocation.methodCallId,
+      SetPublicAssetResponse.deserialize
     )?.created?[generateCreateId];
 
-    if (publicAsset == null) {
-      throw const CannotCreatePublicAssetException();
+    final notCreatedError = response.parse<SetPublicAssetResponse>(
+      invocation.methodCallId,
+      SetPublicAssetResponse.deserialize
+    )?.notCreated?[generateCreateId];
+    
+    if (notCreatedError == null && publicAsset != null) return publicAsset;
+
+    if (notCreatedError?.type == ErrorMethodResponse.overQuota
+      || notCreatedError?.description?.contains('Exceeding public asset quota limit') == true
```

Comment:
IMO, we should only check `type` not `description` to avoid unwanted errors later.

Reply:
Done